auth_oauth2/static/src/js/auth_oauth2.js

```diff
@@ -58,4 +58,34 @@ openerp.auth_oauth2 = function(instance) {
         },
     });
 
+    instance.web.OauthWebClient = instance.web.WebClient.extend({
+    
+        bind_hashchange: function() {
+            var self = this;
+            $(window).bind('hashchange', this.on_hashchange);
+
+            var state = $.bbq.getState(true);
+            if (state.hasOwnProperty('login,password')) {
+                delete state['login,password'];
+            }
```

Comment:
it's look like you only add those four lines, you may move it to the top and call super:

```javascript
var state = $.bbq.getState(true);
if (state.hasOwnProperty('login,password')) {
    delete state['login,password'];
}
this._super();
```

what do you think?